### DISCUSSION TITLE
Just a big thank you

### DISCUSSION THREAD

**Discussion by fabschurt** [Wansmer/treesj#154] [General]
Hi @Wansmer, sorry if this post generates unwanted/useless noise, but I just wanted to thank you fondly for this plugin, as it works flawlessly on my side (leveraging the power of tree-sitter was the smartest move IMO). Splitting/joining is a _huge_ (though overlooked) part of my daily burden as a developer, and neat plugins like this one make my life so much easier.

So, again: thank you so much! 🎉

Cheers

**Comment by Wansmer** [comment]
Thank you so much for your feedback!

**Comment by BirdeeHub** [comment]
Same. When someone asks, what is one neovim plugin you can uncontroversially always recommend to every developer... Its this one for me.
